bcal.c

```diff
@@ -621,6 +621,68 @@ bool lba2chs(char *lba, t_chs *p_chs)
 	return TRUE;
 }
 
+/* Convert any unit in bytes */
+maxuint_t unitconv(char *num,char *unit)
+{
+
```

Comment:
Please remove this blank line. I follow the Linux kernel coding style for C.

Find it here:
https://01.org/linuxgraphics/gfx-docs/drm/process/coding-style.html